src/utilities/common/errorHandling.ts

```diff
@@ -1,5 +1,5 @@
 import { ExecutionResult } from 'graphql';
 
-export function graphQLResultHasError(result: ExecutionResult): boolean {
+export function graphQLResultHasError(result: ExecutionResult<any>): boolean {
```

Comment:
We could avoid having to use `any` here by making this function generic:

```suggestion
export function graphQLResultHasError<T>(result: ExecutionResult<T>): boolean {
```

Reply:
We don’t really use the type parameter, so maybe we can also use `unknown`